src/ui/components/FullModal/FullModal.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from "classnames";
+import { ReactNode } from "react";
+
+import { Modal } from "@src/ui/components/Modal";
+
+import "./full-modal.scss";
```

Comment:
I'd split full modal styles to not import all the rules or use material-ui approach you've already added.
I think it's related to all content components for modal.

Reply:
Related to  #195